test_stubdefaulter.py

```diff
@@ -14,6 +14,8 @@ def f(x=0, y="y", z=True, a=None):
     pass
 def more_ints(x=-1, y=0):
     pass
+def ints_as_hexadecimals(x=0x7FFFFFFF):
```

Comment:
```suggestion
def ints_as_hexadecimals(x=0x7FFFFFFF, y=0b1101, z=0o744):
```
Let's test the rest too for completeness (needs similar changes below).